index.js

```diff
@@ -0,0 +1,130 @@
+
+const URL = "https://trektravel.herokuapp.com/trips"
+
+
+const loadTrips = () => {
+
+  tripList = $('#trip-list');
+  tripList.empty();
+
+
+  axios.get(URL)
+    .then((response) => {
+      response.data.forEach((trip) => {
+        tripList.append(`<li id="${trip.id}"><a href="#">${trip.name}</a></li>`);
+        $("#" + trip.id).click( () => {
+          loadDetails(trip.id);
+          createForm(trip.id, trip.name) })
+
+
+        })
+        tripList.prepend("<h2>All Trips</h2>")
+        $(".current-trips").css("border-style", "solid");
+    })
+    .catch((error) => {
+      console.log(error);
+    });
```

Comment:
This is a good start to error handling, but it doesn't give the user any indication of what's gone wrong.